modules/custom/az_paragraphs/src/Plugin/paragraphs/Behavior/AZTextWithMediaParagraphBehavior.php

```diff
@@ -236,6 +236,11 @@ public function preprocess(&$variables) {
       HTML::getClass($config['style']),
     ];
 
+    // If this paragraph is full-width, add the full-width library.
+    if (isset($config['text_background_full_width'])) {
```

Comment:
I think it'd be better to check for the actual value here.
```suggestion
    if (isset($config['text_background_full_width']) && $config['text_background_full_width'] === 'full-width-background') {
```